<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageStudent</class>
 <widget class="QDialog" name="ManageStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1356</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生信息管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>所属班级: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="11">
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="pushButton_refresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>所属年级: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="pushButton_insert">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox_grade">
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="12">
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="comboBox_class">
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QPushButton" name="pushButton_update">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>学生编号: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_stuID"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>学生姓名: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QLineEdit" name="lineEdit_stuName"/>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>年龄: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="lineEdit_age"/>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>性别: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QComboBox" name="comboBox_sex">
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>联系电话: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>家庭住址: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="5">
      <widget class="QLineEdit" name="lineEdit_adress"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_phone"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>ManageStudent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1297</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
